wiv.js

```diff
@@ -102,8 +97,22 @@ function wiv(params) {
   /**
   Represents the logic to draw a single frame. Animates all wivs
   */
-  function drawLines(canvas, speed, height, tightness, thickness, increment, frame, color, ctx = null) {
-    if(ctx === null){
+  function drawLines(canvas, params) {
```

Comment:
Like @ndresselhaus mentioned in slack, we might want to use JS style named parameters here `{ speed = 1, height = 10, tightness = 1, thickness = 2, increment, frame, color, ctx = null } = {}` and set the appropriate defaults

It has the same end result but the named parameters syntax is pretty sweet and makes it very obvious what is in the params object

Reply:
Assuming this has the same level of support as default parameter values (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters), this would totally break IE support until we add something like babel to the build process. So far, I don't think we're using any features that aren't supported by at least IE 11.